addons/website/static/src/client_actions/configurator/configurator.scss

```diff
@@ -300,6 +300,10 @@
             .o_configurator_feature_status {
                 @include o-position-absolute($card-spacer-y * 0.5 , $card-spacer-x * 0.5);
             }
+
+            .h4 {
+                font-size: 0.875rem;
+            }
```

Comment:
That doesn't really makes sense.
It means that on the configurator, a h4 will be tinier than a h5.
It's surely a no-go to modify a specific heading size like that, especially in a master imp (arguably could have been somethign in a stable version if it was related to views to bypass the need of a -u)

Why don't you replace the `h4` by `h5` to achieve something similar?

Reply:
@rdeodoo I already tried `h5` earlier but the font size is bigger and we don't achieve the desired result.
I have checked with `small`, as before v16.3 we are using `small`, but personally font-size looks too small to me, hence I have changed the heading size `h4`.

Check this out, **configurator card title when using `h5`**
![h5_conf](https://github.com/odoo/odoo/assets/84001602/3e11421b-4e10-4b7e-bb27-a26d4a03b9fd)

**configurator card title when using `small`**
![small_conf](https://github.com/odoo/odoo/assets/84001602/95941234-ff33-4307-8731-81898e9720b7)

If @pierre-pde is okay with `small` I will use it instead of changing the `h4` heading size.

Reply:
Only 5% of the human being are speaking English as main language, let's not fine tune this to pixel perfect solution while the problem will remain for the rest of the planet (Maybe I am exaggerating a bit as it will also have the same improvement for other language but most languages are longer than English anyway and will fit on 2 lines regardless of your change).

Please try to find a solution which would be almost the same as your `0.95rem` using only `h1-6` heading and/or `.h1-6` classes and/or `.small`.

Reply:
Hi @rdeodoo , agree to disagree. 
I'm happy if we can not pixel perfect but make english good.
.small is good

Reply:
>  agree to disagree.

But you agree on everything I said ^^: using existing tools to make english good but not pixel perfect with overkill code

Reply:
@rdeodoo 
used `.small` instead of changing the font-size of `h4`.